packages/cli/src/utils/changelog.js

```diff
@@ -1,49 +1,72 @@
-const _ = require('lodash');
 const path = require('path');
 
 const { readFile } = require('./files');
+const { getMetadata } = require('./metadata');
+
+const isAppMetadata = (obj) =>
+  obj?.app_change_type && obj?.action_key && obj?.action_type;
+
+const isIssueMetadata = (obj) => obj?.app_change_type && obj?.issue_id;
+
+// Turns an empty array into undefined so it will not be present in the body when sent
+const absentIfEmpty = (arr) => (arr.length === 0 ? undefined : arr);
 
 const getChangelogFromMarkdown = (version, markdown) => {
   const lines = markdown
     .replace(/\r\n/g, '\n')
     .replace(/\r/g, '\n')
     .split('\n');
 
-  let startingLine = _.findIndex(lines, (line) =>
-    RegExp(`## .*${version}`).test(line)
+  let startingLine = lines.findIndex((line) =>
+    RegExp(`^#{1,4} .*${version.split('.').join('\\.')}`).test(line)
```

Comment:
There was a change to use `version.replaceAll()` - however this functionality was introduced in Node 15. We're still using Node 14 in CI so tests were failing.

Reply:
Good catch!